railties/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+*   Respond /rails/info/properties.json
```

Comment:
I think this can be summarized a little better. How does this sound?
```
Add JSON support to rails properties route (`/rails/info/properties`).
Now, `Rails::Info` properties may be accessed in JSON format at `/rails/info/properties.json`.
```

Reply:
Thanks for your comment. I've fixed it which follows your opinion.